### DISCUSSION TITLE
How mediaparams on model works?

### DISCUSSION THREAD

**Discussion by imknight** [area17/twill#851] [Q&A]
when I attach an image to a model, it seems like it make use of js ( VUE ? ) and calculates the size of different crop of image, the only ratio will work, the rest of parameter like height and width doesn't work?

<img width="481" alt="Screenshot 2019-10-10 at 3 10 11 PM" src="https://user-images.githubusercontent.com/77604/66546663-156f7000-eb70-11e9-9a9a-760bed1f1ca4.png">

**Comment by Insperio** [comment]
It works the same for me. You could try removing the ratio and then height/width will work fine but obviously you lose the other functionality. Not sure if it's a bug or a feature :)

**Comment by titantwentyone** [comment]
If I remove the ratio, the minValues are still not adhered to. I noticed that the values of the crop turn red (bottom left) if the size is below that specified but the crop is not limited to these values.

**Comment by titantwentyone** [comment]
https://github.com/fengyuanchen/cropperjs/issues/254 could be a potential fix?
